### ISSUE TITLE
Missing functions or build issue

### ISSUE THREAD

**Issue by RedyJack** [twistedfall/opencv-rust#506]
Build project that uses only this:
```
use opencv::prelude::*;
use opencv::core;
use opencv::objdetect;

pub fn detect_faces() {
    let face_cascade = objdetect::CascadeClassifier::new("models/haarcascade_frontalface_default.xml").unwrap();
}
```
results in a lot of missing functions [build.log](https://github.com/twistedfall/opencv-rust/files/13040825/build.log) :
```
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_new_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_delete` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_len_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_isEmpty_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_capacity_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_shrinkToFit` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_reserve_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_remove_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_swap_size_t_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_clear` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_get_const_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_push_const_KeyPoint` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_insert_size_t_const_KeyPoint` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLcv_KeyPointG_set_size_t_const_KeyPoint` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_new_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_delete` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_len_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_isEmpty_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_capacity_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_shrinkToFit` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_reserve_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_remove_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_swap_size_t_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_clear` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_get_const_size_t` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_push_const_int` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_insert_size_t_const_int` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_set_size_t_const_int` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_clone_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_data_const` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_dataMut` in module `$crate::sys`
error[E0425]: cannot find function `std_vectorLintG_inputArray_const` in module `sys`
error[E0425]: cannot find function `std_vectorLintG_outputArray` in module `sys`
error[E0425]: cannot find function `std_vectorLintG_inputOutputArray` in module `sys`
```
Are they really missing or there may be some other issue? In .../sys directory there are some similar functions to deal with vectors but not those exact. If they are missing is there a plan to add them? If not any directions on adding those?

1. Operating system
```
NAME="Ubuntu"
VERSION="20.04.6 LTS
```
2. Open CV installed using apt-get install libopencv-dev 
3. Open CV version
```
cat /usr/lib/x86_64-linux-gnu/pkgconfig/opencv4.pc 
# Package Information for pkg-config

prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib/x86_64-linux-gnu
includedir_old=${prefix}/include/opencv4/opencv
includedir_new=${prefix}/include/opencv4

Name: OpenCV
Description: Open Source Computer Vision Library
Version: 4.2.0
Libs: -L${exec_prefix}/lib/x86_64-linux-gnu -lopencv_stitching -lopencv_aruco -lopencv_bgsegm -lopencv_bioinspired -lopencv_ccalib -lopencv_dnn_objdetect -lopencv_dnn_superres -lopencv_dpm -lopencv_highgui -lopencv_face -lopencv_freetype -lopencv_fuzzy -lopencv_hdf -lopencv_hfs -lopencv_img_hash -lopencv_line_descriptor -lopencv_quality -lopencv_reg -lopencv_rgbd -lopencv_saliency -lopencv_shape -lopencv_stereo -lopencv_structured_light -lopencv_phase_unwrapping -lopencv_superres -lopencv_optflow -lopencv_surface_matching -lopencv_tracking -lopencv_datasets -lopencv_text -lopencv_dnn -lopencv_plot -lopencv_ml -lopencv_videostab -lopencv_videoio -lopencv_viz -lopencv_ximgproc -lopencv_video -lopencv_xobjdetect -lopencv_objdetect -lopencv_calib3d -lopencv_imgcodecs -lopencv_features2d -lopencv_flann -lopencv_xphoto -lopencv_photo -lopencv_imgproc -lopencv_core
Libs.private: -ldl -lm -lpthread -lrt
Cflags: -I${includedir_old} -I${includedir_new}
```
4. rustc 1.72.1 (d5c2e9c34 2023-09-13)

**Comment by twistedfall** [comment]
That looks like another issue https://github.com/twistedfall/opencv-rust/issues/479 which I was not able to reproduce until now, but based on the image from you other issue I can see the error now so expect a fix soon!

**Comment by twistedfall** [comment]
This should be fixed in 0.85.3
